src/main/resources/errors.yaml

```diff
@@ -157,7 +157,7 @@ request-errors:
   # NOTE: UNKNOWN_TABLE_COLUMNS is also in the FILTER scope
   - scope: DOCUMENT
     code: UNKNOWN_TABLE_COLUMNS
-    title: Unknown table columns in document
+    title: Columns in documents are not defined in the table schema
```

Comment:
should it be "Columns in document" (not plural since exceptions are per-document/row).
Also: do we talk about "documents" with tables as well? Could just leave out document here altogether ("Columns are not defined in the table schema").